tests/integration/listview/listview_core.js

```diff
@@ -25,7 +25,7 @@ asyncTest( "The page should be enhanced correctly", function() {
 	expect( 3 );
 	$.testHelper.pageSequence( [
 		function() {
-			$.mobile.changePage( "#basic-linked-test" );
+			$( ".ui-pagecontainer" ).pagecontainer( "change", "#basic-linked-test" );
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?